src/Resources/contao/library/Contao/Config.php

```diff
@@ -513,6 +513,7 @@ protected static function loadParameters()
 		}
 
 		$GLOBALS['TL_CONFIG']['debugMode'] = $container->getParameter('kernel.debug');
+		$GLOBALS['TL_CONFIG']['disableIpCheck'] = $container->getParameter('contao.security.disable_ip_check');
```

Comment:
You can move this to the array above now that we don't need inversion. Dunno why `debugMode` is not up there…